pkgs/desktops/deepin/library/deepin-pdfium/default.nix

```diff
@@ -28,13 +31,16 @@ stdenv.mkDerivation rec {
   dontWrapQtApps = true;
 
   buildInputs = [
+    freetype
+    icu
     libchardet
+    libjpeg
     lcms2
     openjpeg
   ];
 
   meta = with lib; {
-    description = "development library for pdf on deepin";
+    description = "Development library for PDF on Deepin";
```

Comment:
```suggestion
    description = "Development library for PDF on deepin";
```

Reply:
Taken from https://github.com/linuxdeepin/deepin-pdfium.

I will correct it here.

Reply:
According to the official [wiki](https://wiki.deepin.org/zh/03_%E6%8A%80%E6%9C%AF%E8%A7%84%E8%8C%83/01_%E6%96%87%E6%A1%A3%E8%A7%84%E8%8C%83/%E5%93%81%E7%89%8C%E4%B8%93%E6%9C%89%E5%90%8D%E8%AF%8D%E6%8C%87%E5%AF%BC%E6%96%B9%E9%92%88), we should always use lowercase "deepin"